zenodo/modules/deposit/static/json/zenodo_deposit/deposit_form.json

```diff
@@ -76,6 +76,10 @@
         "title": "Publication type",
         "condition": "model.upload_type == 'publication'",
         "titleMap": [
+          {
+            "value": "annotationcollection",
+            "name": "Annotation Collection"
```

Comment:
For consistency, we follow British English capitalization, i.e. only the first word for titles (e.g. `Annotation collection`, `Data management plan`). This should be applied to the other files of the PR as well